src/callbacks_step/save_restart.jl

```diff
@@ -105,6 +105,10 @@ function (restart_callback::SaveRestartCallback)(integrator)
         end
 
         save_restart_file(u_ode, t, dt, iter, semi, restart_callback)
+        if integrator.opts.adaptive
```

Comment:
As I understand, every defined integration scheme in `src/time_integration` has always `integrator.opts.adaptive = false` and hasn't field `integrator.opts.controller`. So it will be always `if false` and no controller info will be saved, I think it's a right behavior in tis case

Reply:
OK, I just wanted to confirm that there are no "type whatever has no field adaptive" errors. Our own schemes are not adaptive, so yes, this is the correct behavior 👍

Reply:
I didn't get an error, i.e. with `Trixi.SimpleSSPRK33()`